Makefile

```diff
@@ -231,7 +231,7 @@ endif
 local-dev-up-with-clair: local-dev-up
 	$(DOCKER_COMPOSE) up -d clair-db
 	$(DOCKER) exec -it clair-db bash -c 'while ! pg_isready; do echo "waiting for postgres"; sleep 2; done'
-	$(DOCKER_COMPOSE) up -d clair
+	DOCKER_USER="$$(id -u):0" $(DOCKER_COMPOSE) up -d clair
```

Comment:
This might be specific to mac. If you don't add this then when you mount a config volume it causes a permission denied error as the user inside the container is different from the user who owns the files in the volume. This is already  being used when starting the quay container https://github.com/quay/quay/blob/master/Makefile#L208